parts/linux/cloud-init/artifacts/sync-tunnel-logs.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/bash
+
+SRC=/var/log/containers
+DST=/var/log/azure/aks/pods
+
+# Install inotify-tools if they're missing from the image
+{{- if IsMariner}}
+command -v inotifywait >/dev/null 2>&1 || dnf install -y inotify-tools
```

Comment:
think you still need this in artifacts/mariner/cse_install_mariner.sh to get it in the VHD? 

if possible, would be nice to use /etc/os-release and vanilla bash vars instead of templates here, but not a big deal either way (trying to reduce reliance on template vars in scripts where possible)

Reply:
This check should be useful in the case where something happens to cause the CSE data to be repaved without the image being updated. @phealy 

I'll look into avoiding templating here.

Reply:
I think just keep the templates for now, not a big deal and simple

Reply:
Just pushed changes for using /etc/os-release instead lol, can you revert if you think it's simpler to use templates - the only concern I have for sourcing from /etc/os-release is if there are already variables with the same name, they will be overwritten from what I understand. Was go and start and testing against that.

Reply:
haha I apologize, let's run with it, I just felt bad for nit picking :P